.changeset/sour-yaks-jog.md

```diff
@@ -0,0 +1,15 @@
+---
+"@comet/brevo-api": minor
+---
+
+Add and export `BrevoTransactionalMailsService` that can be used in the application for sending transactional mails.
+
+**Example Usage of `BrevoTransactionalMailsService`**
+
+```typescript
+constructor(private readonly brevoTransactionalMailsService: BrevoTransactionalMailsService) {}
+
+async send(email: string, htmlContent: string, subject: string): Promise<void> {
+    await this.brevoTransactionalMailsService.send({ to: [{ email }], htmlContent, subject }, data.scope);
```

Comment:
In the code htmlContent is calles textContent, which one is correct? Does is have to be html?

Reply:
This is connected to the other concern, see below

Reply:
Emails can be sent with html or text or both.

Text often was used as the preview in email clients. Afaik all new email clients now also support htmlContent as preview. 
But in general you can set both, but I guess most projects will use htmlContent, unless its just some reporting where text is sufficient.